test/solve-lib/DeFi/FlashLiquidityLib.sol

```diff
@@ -120,6 +124,12 @@ contract FlashLiquidityLib {
         returnObjsFromPull[1] = ReturnObject({returnvalue: abi.encode(true)});
         returnObjsFromPull[2] = ReturnObject({returnvalue: ""});
 
+        // calculate amount out on solver side to compare return value
+        // balanceOfDai = balanceOfDai + (10 * 1e18) + (liquidity0 * 1e18);
+        // balanceOfWeth = balanceOfWeth + (liquidity1 * 1e18);
+        // uint256 amountOut = ((10 * 1e18) * balanceOfWeth) / balanceOfDai;
+        // returnObjsFromPull[2] = ReturnObject({returnvalue: abi.encode(amountOut)});
```

Comment:
Do we need this commented code?

Reply:
We might add check for amount out eventually